IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormPullTests.cs

```diff
@@ -207,6 +207,35 @@ public void Should_remember_user_choice_upon_pull(bool autoStashChecked)
                 AppSettings.PullAction.Merge);
         }
 
+        [Test]
+        public void Should_disable_enable_merge_options_when_no_tracking()
+        {
+            RunFormTest(
+                form =>
+                {
+                    /* Since we're using _commands.StartPullDialog, later FormPullLoad gets called and
+                       in it we call DisableMergeOptionsWhenCurrentBranchIsNotTracking(),
+                       so no need to call it again after the accessor initialization.
+                       Thus, control states are:
+                       Rebase.Enabled = false
+                       Merge.Enabled = false
+                       Fetch.Enabled = true
+                       Fetch.Checked = true
+                    */
+                    var accessor = form.GetTestAccessor();
+
+                    accessor.Merge.Enabled.Should().Be(false);
+                    accessor.Merge.Checked.Should().Be(false);
+                    accessor.Rebase.Enabled.Should().Be(false);
+                    accessor.Rebase.Checked.Should().Be(false);
+                    accessor.Fetch.Enabled.Should().Be(true);
+                    accessor.Fetch.Checked.Should().Be(true);
+                },
+                null, null,
+                //// select an action different from None/fetch
+                AppSettings.PullAction.Merge);
```

Comment:
I guess this argument suppresses the interactive dialog. It might be one of the toolbar dropdown actions.

Reply:
I set it to PullAction.None, maybe this would help.